README.md

```diff
@@ -131,7 +137,9 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An entity relationship diagram (ERD) is a tool that programmers use to  create, and describe data structures and their relationships. We create them for applications to be able to visualize and plan out the entity structures' attributes and actions.
+
+Using the example above, a Genie and Person serve as the relationship of an entity to a Lamp and Pet, respectively. In terms of how these relationships would be mapped on an ERD diagram, one could say that a Person can have many pets but a pet can not have more than one person. In this context, the Person provides the overall scope for which the pet is contained. In the Genie example, since the Genie can pop out of and go inside a lamp, a diamond shaped box would be used in an ERD diagram to describe the action of the Genie in relation to the lamp.
```

Comment:
:+1:

Not sure about the diamond shape, but the Genie belongs to the Lamp or the Lamp has one Genie.